### DISCUSSION TITLE
RunTime is not a registered object

### DISCUSSION THREAD

**Discussion by RWTHLHK** [idaholab/moose#23285] [Q&A Modules: Phase field] [answered]
I am a new user to MOOSE and am running the example of phase-field module. But at 2nd step make a faster model in block Postprocessors, it always reports error RunTime is not a registered object. After I modify the Makefile All_MODULES=yes, the error still occurs.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Could you please paste the error you are getting exactly?
In s2_fasttest.i the Postprocessors do not have a RunTime object
```
[Postprocessors]
  [./evaluations]           # Cumulative residual calculations for simulation
    type = NumResidualEvaluations
  [../]
  [./elapsed]
    type = PerfGraphData
    section_name = "Root"
    data_type = total
  [../]
[]

```

Guillaume
